WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -205,6 +205,62 @@
         <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| "#{c[:name]} (#{c[:iso_code]})" }, value_method: lambda { |c| c[:iso_code] }, disabled: disable_money_input  %>
         <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
 
+        <script>
+          var bandFee = 0;
+          var baseFee = 0;
+          var compLimit = null;
+          var tempLimit = null;
```

Comment:
What does this `tempLimit` variable do?

Reply:
Before disabling the competitor limit, I wanted to store it in a variable so that when it is enabled again, the dues field can immediately change back to displaying the locale with a specific competitor limit.